base_product_mass_addition/models/product_product.py

```diff
@@ -13,25 +13,59 @@ class ProductProduct(models.Model):
 
     qty_to_process = fields.Float(
         compute="_compute_process_qty",
-        inverse="_inverse_set_process_qty",
+        inverse="_inverse_quick_vals",
         help="Set this quantity to create a new line "
         "for this product or update the existing one.",
     )
+    quick_uom_category_id = fields.Many2one(
+        "uom.category", compute="_compute_quick_uom_category_id"
+    )
+    quick_uom_id = fields.Many2one(
+        "uom.uom",
+        domain="[('category_id', '=', quick_uom_category_id)]",
+        compute="_compute_purchase_quick_uom_id",
```

Comment:
remove the word "purchase" here

Reply:
done, pls check https://github.com/akretion/purchase-workflow/pull/10/files#diff-396b594fa7ea09b812220a26448a863eb7d4a6a8da9415464d181da399edb6cdR27 for compatibility with eventual sale modules

Reply:
related would mean we always use uom_po_id